includes/Frontend/MyAccount/BecomeVendor.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace WeDevs\Dokan\Frontend\MyAccount;
+
+/**
+ * Dokan Become Vendor Class
+ *
+ * @since DOKAN_LITE_SINCE
+ *
+ * @package dokan
+ */
+class BecomeVendor {
+    /**
+     * Class Constructor.
+     *
+     * @since DOKAN_LITE_SINCE
+     */
+    public function __construct() {
+        $this->init_hooks();
+    }
+
+    /**
+     * Init Hooks Method.
+     *
+     * @since DOKAN_LITE_SINCE
+     *
+     * @return void
+     */
+    public function init_hooks() {
+        add_action( 'woocommerce_after_my_account', [ $this, 'render_become_a_vendor_section' ] );
+        add_action( 'woocommerce_account_account-migration_endpoint', [ $this, 'load_customer_to_vendor_update_template' ] );
```

Comment:
as discussed add a `plugins_loaded` hooks and remove existing hooks if user is using old version of Dokan Pro

Reply:
Bhai, removed related Dokan Pro hooks by `init` hook instead of `plugins_loaded` hook after discussing with you.